src/applications/caregivers/containers/IntroductionPage.jsx

```diff
@@ -23,62 +19,86 @@ const IntroductionPage = ({ route, router }) => {
+              caregiver applicant will need to provide specific information.
+              You’ll need:
             </p>
-
             <ul>
               <li>Address</li>
-              <li>Phone Number</li>
+              <li>Telephone number</li>
+              <li>Date of birth</li>
               <li>Email address</li>
-              <li>Social Security number or tax identification number.</li>
-              <li>Health care coverage information</li>
+              <li>Social Security Number or Tax Identification Number</li>
+            </ul>
+
+            <p>You will also need:</p>
+            <ul>
+              <li>The VA medical center where the Veteran will receive care</li>
+              <li>
+                Health insurance information for the Primary Family Caregiver
+              </li>
             </ul>
 
             <div>
               <h4 className="vads-u-font-size--h6">
-                What if I need help filling out my application?
+                Where can I get help filling out the form and answers to
+                questions?
               </h4>
-              <p>
-                An accredited representative, like a Veterans Service Officer
-                (VSO), can help you fill out your claim.
-                <a
-                  href={links.caregiverHelp.link}
-                  target="_blank"
-                  rel="noopener noreferrer"
-                >
-                  {links.caregiverHelp.label}
-                </a>
-              </p>
+
+              <span>
+                You may use any of the following to request assistance:
+              </span>
+
+              <ul>
+                <li>
+                  Ask VA to help you fill out the form by calling us at
+                  {''}
+                  <a href={links.VAHelpLine.label}> 877-222-VETS (8387)</a>
+                </li>
+                <li>
+                  Locate and contact the Caregiver Support Coordinator at your
+                  nearest VA health care facility. You may use the online {''}
+                  <a
+                    href="https://www.caregiver.va.gov/support/New_CSC_Page.asp"
+                    target="_blank"
+                    rel="noopener noreferrer"
+                  >
+                    Caregiver Support Coordinator locator
+                  </a>
+                </li>
+                <li>
+                  Contact the VA National Caregiver Support Line by calling
+                  <a
+                    href="tel:8552603274"
+                    target="_blank"
+                    rel="noopener noreferrer"
+                  >
+                    855-260-3274
+                  </a>{' '}
+                  or a Veterans Service Organization
+                </li>
+              </ul>
+
+              <a
+                href="https://www.va.gov/disability/get-help-filing-claim/"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
```

Comment:
I don't think internal links need a domain, or `target` and `rel` attributes.

Reply:
Fixed all occurrences

Reply:
So I removed all of these them the linter was complaining about not having it with target="_blank" so I went ahead and added it back so I could commit.